pyaedt/modules/SolveSetup.py

```diff
@@ -787,14 +779,33 @@ class Setup3DLayout(object):
     setupname: str, optional
         Name of the setup. The default is ``"MySetupAuto"``.
     isnewsetup: bool, optional
-      Whether to create the new setup from a template. The default is ``True.``
-      If ``False``, access is to the existing setup.
+        Whether to create the new setup from a template. The default is ``True.``
+        If ``False``, access is to the existing setup.
     
     """
 
     @property
     def omodule(self):
-        """Analysis module."""
+        """Analysis module.
+        
+        Parameters
+        ----------
+        parent: str
+           AEDT module for analysis setup.
+        solutiontype: 
+            Type of the setup.
+        setupname: str, optional
+            Name of the setup. The default is ``"MySetupAuto"``.
+        isnewsetup: bool, optional
+            Whether to create the new setup from a template. The default is ``True.``
+            If ``False``, access is to the existing setup.
+        
+        Returns
+        -------
+        type
```

Comment:
We will one day pass actual `:class:` return types to allow for cross linking between methods.  Waiting for #176 to be merged for this refactor to start.